src/Car.scala

```diff
@@ -0,0 +1,26 @@
+
+//class represents a car
+class Car {
+
+  var wheels = 4
+  var engine = 0
+  var gas = 0
+  var seats = 0
+
+  def currentCar(): Unit ={
+    println(s"You have $wheels wheels on your car")
+    println(s"You have $gas gallons in your car")
+    println(s"Your engine is $engine liters")
+    println(s"You have $seats in your car")
+  }
+
+  def carDetails(wheels: Int, engine: Int, gas: Int, seats: Int) {
+    println(s"You have $wheels wheels on your car")
+    println(s"You have $gas gallons in your car")
+    println(s"Your engine is $engine liters")
+    println(s"You have $seats in your car")
```

Comment:
You've repeated code here. Why not just call currentCar() inside carDetails().

Then, you could utilize the case class which auto creates all your accessors and mutators.  Change your class car to this:
   case class Car(var wheels: Int = 4, var engine: Int = 0, var gas: Int = 0, var seats: Int = 0) {

And, then change your carDetails() to set each of the fields with the format:  this.wheels = wheels
Plus, replace those println() statements in carDetails() with simply: currentCar().